webservices/resources/legal.py

```diff
@@ -9,8 +9,15 @@
 from webargs import fields
 from pyelasticsearch import ElasticSearch
 from webservices.env import env
+from flask import request
 
 
+if env.get_credential('WRITE_AUTHORIZED_TOKENS'):
+    write_cred = env.get_credential('WRITE_AUTHORIZED_TOKENS') 
+else:
+    write_cred = '12345, abc, def'
```

Comment:
Per our code review session:  Let's change this to an environment variable only so we don't run the risk of introducing an issue with bad credentials letting folks in.